pkg/shell/index.html

```diff
@@ -148,14 +160,20 @@ <h4 class="modal-title" translatable="yes">Authentication</h4>
                 </div>
                 <div class="modal-body modal-scrollable">
                     <table class="listing-ct credential-listing">
-                        <thead>
-                            <tr id="credential-authorize" hidden>
-                                <td translatable="yes" colspan="2">Use my password for privileged tasks and to connect to other machines</td>
+                        <thead class="credential-lock">
+                            <tr>
+                                <td translate colspan="2">Password not usable for privileged tasks or to connect to other machines</td>
+                                <td></td>
+                            </tr>
+                            <tr>
+                                <td translate colspan="2">Reuse my password for privileged tasks and to connect to other machines</td>
```

Comment:
This is the same wording as on the login screen. In addition the string did not change here. So this would be a follow up if a change is needed.